flight/Modules/Airspeed/airspeed.c

```diff
@@ -359,22 +361,25 @@ void baro_airspeedGet(BaroAirspeedData *baroAirspeedData, uint32_t *lastSysTime,
 }
 #endif
 
-static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev)
+static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev, void *ctx, void *obj, int len)
 {
-	AirspeedSettingsData airspeedSettingsData;
-	AirspeedSettingsGet(&airspeedSettingsData);
-	
-	airspeedSensorType=airspeedSettingsData.AirspeedSensorType;
-	gpsSamplePeriod_ms=airspeedSettingsData.GPSSamplePeriod_ms;
+	(void) ev; (void) ctx; (void) obj; (void) len;
```

Comment:
Gotcha.

Since this is boilerplate code, maybe attaching a similarly boilerplate comment might reduce confusion?